docs/operate/build.md

```diff
@@ -178,7 +179,6 @@ The choices are:
 
 1. `Build Loop Follow`
 2. `Build xDrip4iOS`
-3. `Build Glucose Direct`
 4. `Return to Menu`
```

Comment:
`Return to Menu` should now be number `3.`

Reply:
Now Fixed.
Thanks @MikePlante1